helm/values.yaml

```diff
@@ -20,20 +20,24 @@ service:
   type: ClusterIP
   port: 443
 
-# configure access to the notary server
+### VALIDATORS ###
+# validators are a set of configurations (hosts, types, public keys) that can be
+# used for validating one or multiple images (or image signatures). They are tied
+# to their respective image(s) via the image policy below. There are a few handy
+# validators pre-configured.
 validators:
-- name: allow
+- name: allow     # always allow image
```

Comment:
The functionality does not follow from the name, so the comment is misleading.

Also, it's common to use two spaces before inline comments